DFE-Digital/early-careers-framework
# CPDRP-848 CIP only onboarding journey

### Context

https://dfedigital.atlassian.net/browse/CPDRP-848

### Changes proposed in this pull request

- Remove the advisory page before induction programme choice. It was decided that the information on this page is a bit superfluous as it is repeated on the next page where you actually get to make the choice. It also seemed simpler than defining a different set of copy for CIP-only schools.

- Copy updates to the induction programme choice form

- Hide the `full_induction_programme` choice for CIP-only schools.

### Guidance to review

### Testing

Added a new factory trait for cip only schools, tests for the form itself regarding the list of options.

### Review Checks

### How can I view this in a review app?
- Log in as an induction coordinator for a school that hasn't made any choices yet. Check that the first page you are directed to is the induction programme choice page.
- Log in as an induction coordinator for the CIP only school: cip-only-induction-tutor@example.com and compare the copy and options on the page with how it looked for a FIP eligible school.
